### ISSUE TITLE
"The reply address does not match the reply addresses configured for the application" with single sign-on

### ISSUE THREAD

**Issue by ForTheLoveOfD365** [taarskog/crm-powerbi-viewer#36]
Hi,

I found the answer for "The reply address does not match the reply addresses configured for the application" in #20 and #21. However, the solution doesn't seem to work when the page was directed to single sign-on page first.

Any ideas?

Thanks!

**Comment by taarskog** [comment]
Hard for me to investigate with so few details. I'm pretty confident that this is a configuration issue as I do have SSO in one of my own setups and it works as expected.

I suggest you revisit the guide step by step (also make sure you actually have a Power BI Pro license (or better) and that you are able to login directly and see your reports and dashboards.

**Comment by taarskog** [comment]
Reports are smaller tiles - you are sure it is not there (see http://crm-powerbi-viewer.heiigjen.com/assets/images/v1.0/crm-config/config-overview.png)? Can you post a screenshot of the configuration page?

**Comment by ForTheLoveOfD365** [comment]
Hmmm, yesterday the Configuration was displaying "No Reports found." under any workplaces; however,  reports are showing up now... I apologize for the inconvenience and thank you very much for your help on this!!

**Comment by ForTheLoveOfD365** [comment]
Hi Trond,

The IT department decided to rename the D365 URL for security reason. We experienced issues by just updating the URL in the Azure Active Directory, so, we unregistered and re-registered the app. However, the issue of "No Reports Found" happened again, and this time the reports didn't show up the next day (like last time). We went through the all the installation steps to make sure we didn't miss anything.

Any suggestions?

![screenshot](https://user-images.githubusercontent.com/43156216/45643747-9f81fd80-ba70-11e8-9b47-29ebce3b6ca9.png)

Thanks!
